tsdb/query_executor.go

```diff
@@ -739,6 +739,9 @@ func (q *QueryExecutor) writeInto(row *models.Row, selectstmt *influxql.SelectSt
 	// limitedRowWriter and ExecuteAggregate/Raw makes it ridiculously hard to make sure that the
 	// results will be the same as when queried normally.
 	measurement := intoMeasurement(selectstmt)
+	if measurement == "" {
```

Comment:
How can this happen? I though the statement would be normalized by this time, no?